.changeset/sharp-poets-count.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
```suggestion
'@backstage/plugin-scaffolder-backend': minor
```

Reply:
Let's make sure this is a minor bump as the package is version 1.x

Reply:
Done.